### PR TITLE
Fixed no examples in XSLTProcessor `transformToDocument()` method

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/XSLTProcessor/transformToDocument
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/xsltprocessor/transformtodocument/index.md
* Last commit: https://github.com/mdn/content/commit/b71d118ffc6d72b77efad9661110fcc9ede464eb

### CODE REVIEW

File: files/en-us/web/api/xsltprocessor/transformtodocument/index.md [hunk 1]
```diff
@@ -33,6 +33,62 @@ A {{domxref("Document")}}. The actual interface depends on the [output method](h
 
 ## Examples
 
+```html
+<pre id="result"></pre>
```
Comment (T34-active) [inline files/en-us/web/api/xsltprocessor/transformtodocument/index.md:37]:
```suggestion
<div id="result"></div>
```

**Review by T34-active — COMMENTED** [review]
fix wrong element

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
See https://github.com/mdn/content/pull/35431#pullrequestreview-2240690810

File: files/en-us/web/api/xsltprocessor/transformtodocument/index.md [hunk 2]
```diff
@@ -33,6 +33,70 @@ A {{domxref("Document")}}. The actual interface depends on the [output method](h
...
 ## Examples
 
+### Using transformToDocument()
+
+This example demonstrates how to use `transformToDocument()` to transform an XML document using XSLT, resulting in a new XML document structure.
+
+#### HTML
+
+```html
+<div id="result"></div>
```
Comment (Josh-Cena) [inline files/en-us/web/api/xsltprocessor/transformtodocument/index.md:43]:
```suggestion
<pre id="result"></pre>
```

**Review by Josh-Cena — APPROVED** [review]
Looks good :+1: thanks!

File: files/en-us/web/api/xsltprocessor/transformtodocument/index.md (hunk 2 above)
Comment (T34-active) [reply files/en-us/web/api/xsltprocessor/transformtodocument/index.md]: thanks!

**Pull request merged**